### DISCUSSION TITLE
Suggestion to annotate heatmap?

### DISCUSSION THREAD

**Discussion by NataliHB** [ChiLiubio/microeco#67] [Q&A]
Hi Chi,
Thank you so much for this helpful package! I was wondering if anyone had any ideas to annotate further a heatmap created with plot_heatmap. I am trying to include some environmental variables, is there something similar to "annotation_row"? 

Any help is hugely appreciated!

Best,
Natali

**Comment by ChiLiubio** [comment]
Hi Natali

Sorry. I saw this too late. I write my view in the Issues.

Best
Chi
